examples/galaxies/plot_ellipticity.py

```diff
@@ -0,0 +1,123 @@
+"""
+Galaxy Ellipticity Distributions
+================================
+
+This example demonstrate how to sample ellipticity distributions
+in SkyPy.
+
+"""
+
+
+# %%
+# 3D Ellipticity Distribution
+# ---------------------------
+#
+# In Ryden 2004 [1]_, the ellipticity is sampled by randomly projecting
+# a 3D ellipsoid with principal axes :math:`A > B > C`.
+#
+# The distribution of the axis ratio :math:`\gamma = C/A` is a truncated
+# normal with mean :math:`\mu_\gamma` and standard deviation
+# :math:`\sigma_\gamma`.
+#
+# The distribution of :math:`\epsilon = \log(1 - B/A)` is truncated normal
+# with mean :math:`\mu` and standard deviation :math:`\sigma`.
+#
+#
+# In ``SkyPy``, we can sample 3D ellipticities using the
+# :func:`skypy.galaxies.morphology.ryden04_ellipticity()` distribution
+# described above [1].
```

Comment:
This isn't really correct. The `ryden04_ellipticity()` model internally samples the axis ratios of the 3D ellipsoid according to the above and then randomly projects them using [`triaxial_axis_ratio()`](https://skypy.readthedocs.io/en/stable/api/skypy.utils.random.triaxial_axis_ratio.html). So it samples projected 2D axis ratios overall.